app/models/footprint.rb

```diff
@@ -4,4 +4,19 @@ class Footprint < ApplicationRecord
   validates :carbon_in_kg,
             :offset_cost_total,
             :offset_cost_currency, presence: true
+
+  def self.aggregate_footprint_for_month(user_id, month, year)
+    #If we add another service(flight?)
+    #it would have to be added in the the second where method if the flight
+    #table also has a carbon_in_kg column if not this active record
+    #call may have to be redone
+
+    aggregate_footprint = CarMonthlyMileage.joins(:car, :footprint)
+    .select("user_id","carbon_in_kg")
+    .where("LOWER(month) = ?", "#{month.downcase}")
+    .where(year: year, cars: {user_id: user_id})
+    .sum("carbon_in_kg")
+
```

Comment:
Check our miro board to see the setup for this. Since this is in the footprint model, it should start with the Footprint.joints(...)